NFIQ2/NFIQ2Algorithm/src/features/FingerJetFXFeature.cpp

```diff
@@ -289,7 +186,7 @@ FingerJetFXFeature::computeFeatureData(
 	vecRectDimensions.push_back(rect200x200);
 
 	FingerJetFXFeature::FJFXROIResults roiResults = computeROI(
-	    fmds, minCnt, 32, fingerprintImage, vecRectDimensions);
+	    minutiaData, minutiaCount, 32, fingerprintImage, vecRectDimensions);
```

Comment:
The diff would probably read better if `minutiaCount` stayed `minCnt` throughout.